docker-compose.yml

```diff
@@ -372,6 +372,7 @@ services:
 
 volumes:
   postgres-db-volume:
+  postgres-db-bitnami-volume:
```

Comment:
Is it needed to have 2 volumes? If not, please remove one of them.
In general, I'm not sure if lines 373-375 are needed at all.